packages/compass-saved-aggregations-queries/src/components/open-item-modal.tsx

```diff
@@ -135,13 +135,14 @@ const OpenItemModal: React.FunctionComponent<OpenItemModalProps> = ({
   onSubmit,
 }) => {
   return (
-    <ConfirmationModal
+    <FormModal
       open={isModalOpen}
       onCancel={onClose}
-      onConfirm={onSubmit}
+      onSubmit={onSubmit}
       title="Select a Namespace"
-      buttonText="Open"
+      submitButtonText="Open"
       submitDisabled={isSubmitDisabled}
+      scrollClassName={css({ overflow: 'visible' })} // this is so that the selects can hang over the footer and out of the modal
```

Comment:
Here the modal is very short (like ConfirmationModal) but contains a Select which should be able to hang out. So we probably don't want the maxHeight and overflow: auto/scroll behaviour.